source/guides/core-concepts/dashboard-service.md

```diff
@@ -218,7 +218,7 @@ Standard output includes details and summaries of your tests for each spec file
 
 You will also see a summary at the bottom indicating the screenshots, or videos that were uploaded during the recording.
 
-{% img /img/dashboard/standard-output-of-recorded-test-run.png "standard output %}
+{% img /img/dashboard/standard-output-of-recorded-test-run.png "standard output" %}
```

Comment:
So yeah, the absence of that trailing `"` caused these sections to not be generated into the HTML.

I don't know why and I haven't investigated to determine the root cause: I know, I'm being lazy... yeah, I just gotta own that 😳Sorz... kinda; the sun is shining outside and I'm about to head out for a run so I'm not really that sorry 😄

> I only stumbled onto this by accident because I was idly messing about with Pandoc filters to see if they'd help with [this issue](https://github.com/cypress-io/cypress-documentation/issues/861) about converting header sections -- I don't think a Pandoc filter will help with that issue -- and my filter caused the Hexo build to fail because these sections started being picked up.

Reply:
Yes, I've seen this before. Thanks for noticing. 👍